src/GeositeFramework/plugins/layer_selector_v2/config.js

```diff
@@ -0,0 +1,93 @@
+﻿require({
+    packages: [
+        {
+            name: "underscore",
+            location: "//cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3",
+            main: "underscore-min"
+        }
+    ]
+});
+
+define([
+        "dojo/_base/declare",
+        "dojo/json",
+        "use!tv4",
```

Comment:
We were looking at dropping `use` in favor of the dojo shim system that's now available.  Consider upgrading this.